src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/RazorHtmlCSharpLanguageServer.cs

```diff
@@ -34,6 +34,7 @@ internal class RazorHtmlCSharpLanguageServer : IDisposable
             {
                 TraceSource = loggerProvider.GetTraceSource()
             };
+            _jsonRpc.TraceSource = loggerProvider.GetTraceSource();
```

Comment:
Hmmm, so this trace source is different than the line above on 35?

Reply:
It's the same.

Reply:
Updated to make this clearer.